classes/privacy/provider.php

```diff
@@ -34,16 +34,13 @@
  */
 class provider implements \core_privacy\local\metadata\null_provider {
 
-    // To provide php 5.6 (33_STABLE) and up support.
-    use \core_privacy\local\legacy_polyfill;
-
     /**
      * Get the language string identifier with the component's language
      * file to explain why this plugin stores no data.
      *
      * @return  string
      */
-    public static function _get_reason() {
+    public static function get_reason() {
```

Comment:
Hi Eloy,
why not `get_reason(): string` i.e. it looks like _return type declaration_ is missing here?

HTH,
Matteo

Reply:
Ah, absolutely! Good point! I missed which the original purpose of the polyfill was, hah! Amending!

Thanks!

Reply:
Should be fixed now. Ciao :-)